WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -315,7 +315,7 @@ local function ConstructTextEditor(frame)
         level)
       end
     elseif menu == "sizes" then
-      local sizes = {12,14,16,18,20,22,24}
+      local sizes = {10, 12, 14, 16}
```

Comment:
The bigger font sizes felt very excessive. This feels like a more reasonable list of font sizes, with 16 arguably being already unnecessarly big.

Why did you want to have such big sizes?

Reply:
I am running a 27" 1440p monitor with a 0,5 UI scale to have more space overall. The 12 felt fairly low. 16 feels about perfect, maybe 18 max. 

The reason was that I have friends that enjoy bigger font sizes due to bad eyesight and thought it might accommodate more people. In hindsight, 24 is probably way too much. I am very inexperienced with UI/UX and that probably contributed to the excessive sizes.

Reply:
Well, a 24pt editor font size won't help your friends, because the rest of the interface is still using 12pts.